Index: gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemsSearch.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemsSearch.java b/gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemSearch.java
rename from gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemsSearch.java
rename to gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemSearch.java
--- a/gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemsSearch.java	
+++ b/gc-coffee/src/main/java/edu/example/gccoffee/repository/search/OrderItemSearch.java	
@@ -1,4 +1,4 @@
 package edu.example.gccoffee.repository.search;
 
-public interface OrderItemsSearch {
+public interface OrderItemSearch {
 }
